README.md

```diff
@@ -115,9 +115,57 @@ The Cassandra tables in the `tsdb` keyspace (by default) are organized into two
 - **metadata** : tables that enable metadata lookup directly or to enable queries
 - **data** : tables that store the timestamped values
 
-![](docs/tables.drawio.png)
+#### Metadata
 
-To be a bit more explicit, all these tables are written to on ingest.  The "metric_names" table is only read by the "metricNames" metadata query; the "tag_keys" by the "tagKeys" metadata query and the "tag_values" by the "tagValues" metadata query.  The "series_set" and "data_raw" tables are only read by the "query" data query.
+`metric_names`:
```

Comment:
How does the downsampled bool field in metric names table get set?

Reply:
I probably concurrently switched that out on you. I since changed that boolean field to `aggregators`, which is a `set<text>` that'll get populated first with the value "raw" as the raw data is ingested. What I forgot to add this afternoon is the corresponding bit down in the downsample section. I'll add that and it'll connect up those dots.